<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resistance_series_connected_Window</class>
 <widget class="QMainWindow" name="resistance_series_connected_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="input_frame">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="r2_label">
       <property name="font">
        <font>
         <family>Source Code Pro Medium</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>r2   </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="r2_input">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="r1_input">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="r1_label">
       <property name="font">
        <font>
         <family>Source Code Pro Medium</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>r1   </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="resistance_series_connected_result_button">
       <property name="font">
        <font>
         <family>Source Code Pro Medium</family>
         <pointsize>14</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Вычислить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="result_frame">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Результат</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="result_text">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../formula4.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resistance_series_connected_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вычисление сопротивления электрической цепи, состоящей из двух последовательно соединенных сопротивлений</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
